### ISSUE TITLE
datalist options hidden value, show only label

### ISSUE THREAD

**Issue by keyboardbreaker** [space10-community/conversational-form#372]
So I've followed the [docs](https://space10-community.github.io/conversational-form/docs/1.0.0/functionality/autocomplete/) on using autocomplete.

Is there a way to hide the forms inputed value and only display the label.

basically my problem is that both show, when desirably label is all that is needed.

<img width="369" alt="Screenshot 2019-11-22 at 13 19 01" src="https://user-images.githubusercontent.com/6376635/69429236-ffd29600-0d2a-11ea-80ff-8a837453246b.png">

**Comment by aidacarbajo** [comment]
hii, do you find a solution? I have the same problem

**Comment by keyboardbreaker** [comment]
im sorry i didnt find a solution

**Comment by sidverma24** [comment]
Hello, I also faced the same problem but there is no native way. For achieving that.

**Comment by aidacarbajo** [comment]
> Hello, I also faced the same problem but there is no native way. For achieving that.

Yes, I finally did a dirty 'solution' that I really don't like hahaha. I write the emails of the users on the value like this value="{{usuario.email}}" and their id like id="{{usuario.uid}}". So before sending the form I modified the value of the selected option by its id. 

```
var datasetEvento = <HTMLInputElement>document.getElementById("eventoss");  // input
var inputEvento = datasetEvento.value;  // input value
var idEvento = document.querySelector("#eventos option[value='"+inputEvento+"']").id; // id from the selected option of the dataset #eventos
this.datosForm.value.evento = idEvento;
```

**Comment by odaeshbli87** [comment]
<input autocomplete="off">
Write inside the input
autocomplete="off"

**Comment by odaeshbli87** [comment]
<input "autocomplete="off">
